.github/workflows/release.yml

```diff
@@ -52,14 +52,17 @@ jobs:
         run: |
           export PATH=${PATH}:`go env GOPATH`/bin
           go get -v -t -d ./...
-      - name: Install gox
+      - name: Install bingo and deps
         run: |
           export PATH=${PATH}:`go env GOPATH`/bin
-          go get github.com/mitchellh/gox  
+          go get github.com/bwplotka/bingo && go mod tidy
+          bingo get
```

Comment:
You install the one `bingo` dep, then let `bingo` take care of installing other developer tool deps.